<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLogin</class>
 <widget class="QWidget" name="frmLogin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>136</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>293</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app_new.qrc">
    <normaloff>:/app_new/medfisis.ico</normaloff>:/app_new/medfisis.ico</iconset>
  </property>
  <property name="toolTip">
   <string>Login Dialog</string>
  </property>
  <property name="whatsThis">
   <string>This is a login dialog.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbUser">
     <property name="text">
      <string>Username</string>
     </property>
     <property name="buddy">
      <cstring>lineUser</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineUser">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbPasswd">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="buddy">
      <cstring>linePasswd</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="linePasswd">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbFiles">
     <property name="text">
      <string>Database</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbFiles</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbFiles">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushLogin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="icon">
      <iconset resource="app_new.qrc">
       <normaloff>:/app_new/password.png</normaloff>:/app_new/password.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+L</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushLogin</sender>
   <signal>clicked()</signal>
   <receiver>frmLogin</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbFiles</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>frmLogin</receiver>
   <slot>loadForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbFiles</sender>
   <signal>activated(QString)</signal>
   <receiver>frmLogin</receiver>
   <slot>showForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validate()</slot>
  <slot>itemSelected(QString)</slot>
  <slot>loadForm(QString)</slot>
  <slot>showForm(QString)</slot>
 </slots>
</ui>
